doc/source/_static/js/assistant.js

```diff
@@ -30,150 +30,153 @@ chatPopupDiv.innerHTML = `
     </div>
   </div>
   <div class="chatFooter text-right p-2">
-    © Copyright 2023, The Ray Team.
+    © Copyright 2024, The Ray Team.
```

Comment:
is this year change related to AI assistant breaking?

Reply:
a request from @angelinalg per https://github.com/ray-project/ray/issues/42461

Reply:
Usually I do one issue -> one PR. I'll make a separate issue for this if that's preferred.